dandi/consts.py

```diff
@@ -130,3 +130,6 @@
 #: HTTP response status codes that should always be retried (until we run out
 #: of retries)
 RETRY_STATUSES = (500, 502, 503, 504)
+
+external_file_types = [".mp4"]
```

Comment:
I would call it `external_file_extensions` to be clearer since "types" could mean different things

Reply:
yeah thanks that is more descriptive